assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -542,10 +542,8 @@ che.singleport.wildcard_domain.ipless=false
 
 # Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
 # plugins dependencies to a workspace
-che.workspace.plugin_broker.image=eclipse/che-plugin-broker:v0.14.0
-che.workspace.plugin_broker.theia.image=eclipse/che-theia-plugin-broker:v0.14.0
 che.workspace.plugin_broker.init.image=eclipse/che-init-plugin-broker:v0.14.0
-che.workspace.plugin_broker.vscode.image=eclipse/che-vscode-extension-broker:v0.14.0
+che.workspace.plugin_broker.unified.image=eclipse/che-unified-plugin-broker:v0.15.0
```

Comment:
I forgot to include that in my review -- I think moving forward (once the other brokers are gone as standlone) unified might make less sense. We should at least discuss naming on the property.

Reply:
@metlos not sure this would drastically simplify PR, but agree that naming could be improved and that after collapsing all the brokers name unified would be kinda outdated. 
I do not want to use property `che.workspace.plugin_broker.image` because it is already used for another broker and in case ops of Che installation overrode it with a custom (or old) version of that broker it would brake workspaces start after the update to Che which uses unified broker.
I'm OK to change the name though. @metlos @amisevsk do you have ideas on how it should be called?

Reply:
I personally don't have a better name for this. I'm OK with `unified` given the reasons you gave.